modules/gltf/src/lib/extensions/EXT_structural_metadata.ts

```diff
@@ -467,65 +514,67 @@ function getPropertyDataNumeric(
     return [];
   }
 
-  // Single value (not an array)
-  const attributeValueArray: number[] = [];
-  for (let index = 0; index < numberOfElements; index++) {
-    const value = valuesData[index];
-    attributeValueArray.push(value);
-  }
-  return attributeValueArray;
+  return valuesData;
 }
 
+/**
+ * Parse variable-length array data
```

Comment:
This reads very generic, more context for new readers if possible

Reply:
Done